src/main/java/io/confluent/idesidecar/restapi/application/SidecarInfo.java

```diff
@@ -148,4 +157,13 @@ static String getSystemOrEnvProperty(String name) {
     var result = System.getProperty(name);
     return result != null ? result : System.getenv(name);
   }
+
+  public String getUserAgent() {
+   return "Confluent-for-VSCode/v%s (https://confluent.io; support@confluent.io) sidecar/v%s (%s/%s)".formatted(
+        vsCode().map(VsCode::extensionVersion).orElse("unknown"),
+        version(),
+        osType(),
```

Comment:
I think we need to lowercase the `OperatingSystemType` here:
```suggestion
        osType().name().toLowerCase(),
```